packages/terra-action-header/src/orion-fusion-theme/ActionHeaderContainer.module.scss

```diff
@@ -0,0 +1,37 @@
+@import '~terra-mixins/lib/Mixins';
+
+:local {
+  .orion-fusion-theme {
+    // .flex-header
+    --terra-action-header-align-items: center;
+    --terra-action-header-background-color: #f4f4f4;
+    --terra-action-header-border-bottom: 0.16666667rem solid #64696c;
```

Comment:
Updated to `0.16666667rem solid #64696c;` from [`2px solid #9b9fa1;`](https://github.com/cerner/terra-core/pull/2921/files#diff-88dce3fa259e93fb4378f904ec7774c0L6) based on this [conversation](https://github.com/cerner/terra-clinical/pull/641#discussion_r405595342)

Reply:
@emilyrohrbough @jeremyfuksa Technically - Fusion theme was intentionally set to rem value (not pixels) for borders because that is exactly how it is set in actual Fusion.

Reply:
@neilpfeiffer I'm not sure I follow your comment. are you request I revert back to 2px and just update the color?